Probe/src/probe/logger.go

```diff
@@ -37,18 +37,18 @@ type CloudLogger struct {
 
 //TODO(langenbahn): add log/error destinations in config proto
 type probeLog struct {
-	sendTime  string
-	probeType string
-	latency   int
-	state     string
-	region    string
-	token     string
+	SendTime  string `json:"sendTime"`
```

Comment:
nit: Capitalized struct members should have comment, e.g. //SendTime is...

Reply:
addresesd in 024773b